geppetto.js/geppetto-ui/src/3d-canvas/showcase/examples/AuditoryCortexExample2.js

```diff
@@ -0,0 +1,199 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core';
+import Canvas from '../../Canvas';
+import CameraControls from '../../../camera-controls/CameraControls';
+import Button from "@material-ui/core/Button";
+import Loader from "@geppettoengine/geppetto-ui/loader/Loader";
+
+const INSTANCE_NAME = 'acnet2';
+const COLORS = [
+  { r: 0, g: 0.2, b: 0.6, a: 1 },
+  { r: 0.8, g: 0, b: 0, a: 1 },
+  { r: 0, g: 0.8, b: 0, a: 1 },
+  { r: 0, g: 0.8, b: 0, a: 0.5 },
+];
+const SELECTION_COLOR = { r: 0.8, g: 0.8, b: 0, a: 1 };
+
+const styles = () => ({
+  container: {
+    height: '800px',
+    width: '1240px',
+    display: 'flex',
+    alignItems: 'stretch',
+  },
+});
```

Comment:
there should be an empty line between class definitions, need to update the eslint config at some point (#63)